CONTRIBUTING.md

```diff
@@ -11,11 +11,11 @@ If you are unfamiliar with repositories with submodules, please first read this
 Following the gitflow workflow, Rigbox and its main submodules (signals, alyx-matlab, and wheelAnalysis) each have two main branches: the dev branch is where new features are deployed, and the master branch contains the stable build that most users will work with. Contributors should create a new branch for any changes/additions they wish to make. Contributors should then create a pull request for the contributed branch. If making a change to a submodule, a pull request should be sent to that submodule's repository. (e.g. if a user is making a change to a file within the signals repository, a pull request should be made to the [signals repository](https://github.com/cortex-lab/signals/pulls), not to the Rigbox repository.) **All pull requests should be made to the dev branch of the appropriate repository.** All pull requests will be reviewed by the project maintainers. Below are procedural guidelines to follow for contributing via a pull request:
 
 1. Ensure any new file follows [MATLAB documentation guidelines](https://www.mathworks.com/help/matlab/matlab_prog/add-help-for-your-program.html) and is accompanied by a test file that adequately covers all expected use cases. This test file should be placed in the appropriate repository's 'tests' folder, and follow the naming convention of `<newFile>_test`. If the contributor is not adding a new file but instead changing/adding to an exisiting file that already has an accompanying test file, a test that accompanies the contributor's code should be added to the existing test file. See the [Rigbox/tests folder](https://github.com/cortex-lab/Rigbox/tree/dev/tests) for examples.
-2. Ensure all existing tests for the entire repo pass. To do so, in MATLAB within the `Rigbox\tests` folder, run 
-> results = runtests(pwd,'IncludeSubfolders',true)
+2. Ensure all existing tests for the entire repo pass. To do so, in MATLAB within the `Rigbox/tests` folder, run 
```

Comment:
@k1o0 do you want to update this point to mention the CI testing? Contributors shouldn't necessarily have to manually run the tests now?

Reply:
No, I think it's still useful for users to run tests before committing

Reply:
depending on the system they are running tests on though (e.g. different versions of MATLAB with different packages installed), these may fail for non-obvious, which can be really confusing, as we've learned.

Reply:
It doesn't matter, that's still informative.  We should be aware of differences between MATLAB versions, etc. that part of the reason for making tests in the first place.  If the reasons for code failing were always obvious then we wouldn't need tests at all.

Reply:
This should at the least be re-worded. "Ensure" is too strong: if someone contributes code that fails on a system that has MATLAB 2017B, for instance, but that code passes all tests on ZTEST, then they should feel fine opening a PR. They shouldn't have to spend time realizing that the issue is that they're running MATLAB 2017B. We should provide an environment for them to run tests on - is there an easy way to have the CI run on forks, for example? Or at the very least, specify the enviornment for them in this document.

Reply:
I disagree.  We state in the readme that Rigbox requires MATLAB 2017b or later.  If someone makes a change which breaks 2017b, we should know about it and address the problem.  

Again, these scenarios you're coming up with are precisely why tests are important.  Without them bugs creep in and there's absolutely no cost to running tests multiple times or on different machines.  Besides, we wrote 'below are procedural guidelines' which is pretty soft wording already.  If you like you can remove the numbering and make them bullet points instead.  That way there's no explicit order.

Reply:
fine, I'll just make an explicit note then of the environment we use on ZTEST for testing, and that different environments may be the source of errors

Reply:
As you wish but these are things people should check for and raise as GitHub issues, not simply write off as some sort of quirk!

Reply:
yeah agreed, I just want to make sure people are more aware than I was (e.g. w/ `AlyxPanel_test`) that the enivornment may be the source of an error, and that checking that can be a first step in debugging

Reply:
I would be surprised if upon failing the tests they deleted the branch and gave up :P